DOCS: Add extensions documentation (#4850)
ansys/pyaedt: doc/source/User_guide/extensions.rst

Extensions
==========

> PyAEDT extensions provide a simplified interface to perform automated workflows.

Reviewer: ```suggestion
Extensions provide a simplified interface to perform automated workflows in AEDT.
```

> The user can install or uninstall these extensions using the extension manager.

Reviewer: ```suggestion
Extensions can be added or removed from AEDT using the extension manager.
```

Reviewer 2: It should be active voice and second person:
In AEDT, you can use the Extension Manager to install or uninstall extensions.

For more information, see `Extension Manager <https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html#extension-manager>`_.

> Depending on the application, the user can find different extensions, the following sections describe each of them.

Reviewer: ```suggestion
Extensions are generally tool-specific and are therefore only accessible given the appropriate context. The following sections provides further clarification
regarding the context.
```

> Extensions can be launched in standalone mode from the console or from a python script. This is described on each extension section.

Reviewer 2: Again, try to write in active voice, using "you" as the subject.
You can launch extensions in standalone mode from the console or a Python script.

P.S. Python is a proper noun.

Project extensions
==================

> Project extension apply to all extensions that are visible to all kind of AEDT applications.

Reviewer: ```suggestion
Project extension apply to all extensions that are pplicable for all AEDT applications.
```

.. grid:: 2

   .. grid-item-card:: Import Nastran
            :link: pyaedt_extensions_doc/project
            :link-type: doc

>             Import Nastran or STL file in any 3D modeler application.

Reviewer 2: ```suggestion
            Import a Nastran or STL file in any 3D modeler application.
```
